wys/api/python/wys_api.py

```diff
@@ -424,37 +424,35 @@ def update_locations(conn, loc_table):
                        a.loc, a.geom 
                 FROM daily_intersections A
                 LEFT JOIN locations B ON A.api_id = B.api_id
```

Comment:
I'm pretty sure this won't account for new signs, only moved ones. 
There needs to be an `OR api_id NOT IN (SELECT api_id FROM locations) added here